src/test/java/fr/zcraft/quartzlib/tools/mojang/MojangHeadTest.java

```diff
@@ -0,0 +1,28 @@
+package fr.zcraft.quartzlib.tools.mojang;
+
+import fr.zcraft.quartzlib.MockedBukkitTest;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Objects;
+
+class MojangHeadTest extends MockedBukkitTest {
+    @Test
+    void canCreateHeadItems() {
+        Assertions.assertEquals("MHF_Cake", MojangHead.CAKE.getHeadName());
```

Comment:
We should also test this for composed names, to ensure the names are correctly transformed, and ones with a specified name to ensure it is taken into account.

```suggestion
        Assertions.assertEquals("MHF_Cake", MojangHead.CAKE.getHeadName());
        Assertions.assertEquals("MHF_MushroomCow", MojangHead.MUSHROOM_COW.getHeadName());
        Assertions.assertEquals("MHF_CoconutB", MojangHead.COCONUT_BROWN.getHeadName());
```

Reply:
This was just a quick test to cover my changes, but ideally since the MHF heads are a known, fixed list, we should probably test every one of them directly.

Maybe this would be better done in a separate PR however?

Reply:
Yes and yes.